echopop/tests/test_data_transect_analysis.py

```diff
@@ -0,0 +1,633 @@
+import pandas as pd
+import numpy as np
+from echopop.survey import Survey
+
+def test_fit_binned_length_weight_relationship( mock_survey ):
+    
+    #### Pull in mock Survey object
+    objS = mock_survey
+    
+    ### Initialize objS for `length_weight`
+    objS.statistics[ 'length_weight' ] = { }
+    
+    ### Re-parameterize `specimen_df` with dummy data 
+    objS.biology[ 'specimen_df' ] = pd.DataFrame(
```

Comment:
You can specify the type (you'll see the methods and attributes turn yellow, which i think means that VSCode recognize these attributes/methods), and also use mock_survey directly in the test:
```suggestion
def test_fit_binned_length_weight_relationship( mock_survey: Survey ):
    
    ### Initialize objS for `length_weight`
    mock_survey.statistics[ 'length_weight' ] = { }
    
    ### Re-parameterize `specimen_df` with dummy data 
    mock_survey.biology[ 'specimen_df' ] = pd.DataFrame(
```